doc/changelog.rst

```diff
@@ -18,6 +18,8 @@ Enhancements
 ------------
 
 - Added bootstrap estimate of bias (issue 6).
+- Added `bias_corrected` function for jackknife and bootstrap, which computes the bias corrected estimates
```

Comment:
```suggestion
- Added ``bias_corrected`` function for jackknife and bootstrap, which computes the bias corrected estimates.
```

Not certain but I think restructured text uses double backticks for code formatting. Is there an issue number we could add?